span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/fieldgenerators/HttpFieldsGenerator.java

```diff
@@ -255,11 +277,22 @@ void populateOtherFields(Event.Builder eventBuilder) {
     fieldGeneratorMap.put(
         RawSpanConstants.getValue(HTTP_PATH),
         (key, keyValue, builder, tagsMap) -> setPath(builder, tagsMap));
+    fieldGeneratorMap.put(
+        OTelHttpSemanticConventions.HTTP_TARGET.getValue(),
+        (key, keyValue, builder, tagsMap) -> setPath(builder, tagsMap));
 
     // Host Handler
     fieldGeneratorMap.put(
             RawSpanConstants.getValue(HTTP_HOST),
             (key, keyValue, builder, tagsMap) -> builder.getRequestBuilder().setHost(keyValue.getVStr()));
+    fieldGeneratorMap.put(
+        OTelHttpSemanticConventions.HTTP_HOST.getValue(),
+        (key, keyValue, builder, tagsMap) -> builder.getRequestBuilder().setHost(keyValue.getVStr()));
+
+    // set scheme if specified
+    fieldGeneratorMap.put(
```

Comment:
Can we add one more test for this, I think, this is a new addition of setting schema. Mostly, it is set from URL.
So, we need to add a test without a URL to test this?